### PR TITLE
[Mobile] Initial admin panel implemented

### PR DESCRIPTION
**Pull request by smelihportakal**
Admin panel implemented for approving and rejecting created game pages

for issue #545.

### CODE REVIEW

**Comment by umutdmr** [comment]
Can you give me credentials for admin user? I want to test it.

**Comment by smelihportakal** [comment]
> Can you give me credentials for admin user? I want to test it @smelihportakal.

You must connect local database. Then you should create a new user and from you must make isAdmin field true for user. For now you can just view admin panel from drawer.

File: app/mobile/lib/presentation/widgets/drawer_widget.dart [hunk 1]
```diff
@@ -159,6 +160,19 @@ class LoggedDrawer extends StatelessWidget {
...
+                  onTap: () {
+                                // Open a new page with game information
+                                Navigator.push(
+                                  context,
+                                  MaterialPageRoute(
+                                    builder: (context) => AdminPanel(),
+                                  ),
+                                );
+                              },
+                ),
```
Comment (erkamkavak) [inline app/mobile/lib/presentation/widgets/drawer_widget.dart:175]: Don't forget to make this visible to only admin users in the following pr's.

**Comment by erkamkavak** [comment]
Seems pretty good. Admin panel where admins can approve pending games is implemented: 
image
image

Nice job!

**Pull request merged**